compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/InterfaceLowering.kt

```diff
@@ -67,6 +80,33 @@ private class InterfaceLowering(val context: JvmBackendContext) : IrElementTrans
         Visibilities.isPrivate(function.visibility) && function.name != clinitName ||
                 function.origin == IrDeclarationOrigin.FUNCTION_FOR_DEFAULT_PARAMETER ||
                 function.origin == JvmLoweredDeclarationOrigin.SYNTHETIC_METHOD_FOR_PROPERTY_ANNOTATIONS
+
+    private fun IrFunction.isJvmDefault(): Boolean =
+        (hasAnnotation(JVM_DEFAULT_FQ_NAME) ||
+                (this is IrSimpleFunction && correspondingPropertySymbol?.owner?.hasAnnotation(JVM_DEFAULT_FQ_NAME) == true)) &&
```

Comment:
'propertyIfAccessor' could be used here

Reply:
I searched `propertyIfAccessor` and all of them seem to be extensions of some descriptors?

Reply:
Oh...seems we should add one for  IrFunction

Reply:
@max-kammerer Done! Please take a look again.